api/play.json

```diff
@@ -1,10 +0,0 @@
-{
```

Comment:
I removed this file because it is no longer needed, as Django will dynamically generate a response sent to /api/play-list/. This text is in the spec, if we need to refer to it.